components/public-api/gitpod/v1/user.proto

```diff
@@ -11,13 +11,31 @@ option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
 service UserService {
   // GetAuthenticatedUser allows to retrieve the current user.
   rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {}
+
+  // UpdateUser updates the properties of a user.
+  rpc UpdateUser(UpdateUserRequest) returns (UpdateUserResponse) {}
 }
 
 message GetAuthenticatedUserRequest {}
 message GetAuthenticatedUserResponse {
   User user = 1;
 }
 
+message UpdateUserRequest {
+  string user_id = 1;
+  string name = 2;
```

Comment:
should we mark those params `optional`?

Reply:
done. 
only reapeated cannot be optional, but hey, everything is optional by default, so nothing's really changes.